src/class/SimpleData.ts

```diff
@@ -603,6 +604,36 @@ export default class SimpleData {
         return this
     }
 
+    @logCall()
+    addRank({
+        rankTitle,
+        rankBy,
```

Comment:
To stay consistent with the other methods, this should probably be named "key". It's the key storing the values we use to add the rank.